oasis/oasis.py

```diff
@@ -18,7 +18,7 @@
 HTTP_OK = 200
 
 runners = dict()
-lock = threading.Lock()
+lock = tornado.locks.Lock()
```

Comment:
suggestion: better one PR does one thing. But as you have done it, we can keep it in this PR.